components/home/SearchBar.js

```diff
@@ -11,29 +11,50 @@ export default class SearchBar extends React.Component {
   }
 
   getMatches = () => {
-    fetch("http://localhost:8000/books/search_book/" + this.state.search).then(
-      console.log((res) => console.log(res.json()))
-    );
+    fetch("http://localhost:8000/books/search_book/" + this.state.search)
+      .then((res) => res.json())
+      .then((res) => this.setState({ matches: res.data.slice(0, 5) }));
+  };
+
+  goToRecommend = (book_id) => {
+    //navigate to /recommend?book_id=book_id
+    window.location.href = `/recommend?book_id=${book_id}`;
```

Comment:
Maybe we could implement a better navigation/routing system

Reply:
https://nextjs.org/docs/api-reference/next/link